src/startupscriptgenerator/src/node/scriptgenerator.go

```diff
@@ -149,7 +149,6 @@ func (gen *NodeStartupScriptGenerator) GenerateEntrypointScript() string {
 		scriptBuilder.WriteString("if [ -d node_modules ] || [ -L node_modules ]; then\n")
 		// We move the directory/link first to prevent node from start using it
 		scriptBuilder.WriteString("    mv -f node_modules _del_node_modules || true\n")
-		scriptBuilder.WriteString("    nohup rm -fr _del_node_modules &> /dev/null &\n")
```

Comment:
I am wondering if you should revert this change in this PR and just keep the PHP change...i will make this change in a different PR